.azuredevops/appcenter-sdk-unity-build-release-1es.yml

```diff
@@ -4,6 +4,19 @@ trigger:
 pr:
 - master
 - develop
+
+variables: 
+- name: EDITOR_URL_WIN
```

Comment:
Where is this variable used?

Reply:
It used in build.cake script, that running in Install Unity task. 
https://github.com/microsoft/appcenter-sdk-unity/blob/b85e7669d71a38c304646038ad26bc3819ca10c5/build.cake#L377